### DISCUSSION TITLE
Display correlation between or among habits

### DISCUSSION THREAD

**Discussion by LuccoJ** [iSoron/uhabits#128] [Feature Requests]
This is related to and dependent on issue #51 and other ones like #31, but since it isn't quite the same thing, I decided to file it separately.

Sometimes, you want to know whether your sticking to a habit tends to be dependent on fulfilling another habit, or generally how habits are related to one another.

Practical examples:
- Do I tend to overeat on days when I've had more exercise than usual?
- Do I jog past my target distance when I've had coffee?
- Do I get a headache when I go to bed late?

So it would be nice to integrate somewhat advanced statistical correlation graphs into the "combinated statistics" display, if that happens to be implemented.

**Comment by iSoron** [comment]
Thank you for the suggestion. This is a very interesting feature. In addition to computing the correlation between a group of habits that the user has manually picked, the app could also show, in the statistics page for any single habit, what other habits that have the highest/lowest correlation to it. This feature could be implemented even before the other issues are closed.

**Comment by matthewfcarlson** [comment]
I'd love to work on this
